SPECS/golang/golang-1.15.spec

```diff
@@ -22,7 +22,6 @@ URL:            https://golang.org
 Source0:        https://dl.google.com/go/go%{version}.src.tar.gz
 Source1:        https://dl.google.com/go/go1.4-bootstrap-20171003.tar.gz
 Patch0:         go14_bootstrap_aarch64.patch
```

Comment:
This patch is applied in a way that makes the SRPM architecture-dependent. I think we should be good if we just remove the `ifarch` condition around the patch.

Reply:
Agreed. This patch should be fine to apply on X64 architecture as well

Reply:
fixed